public/views/index-template.html

```diff
@@ -1,18 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script nonce="[[.Nonce]]">
-      // https://github.com/GoogleChromeLabs/tti-polyfill
-      !(function () {
-        if ('PerformanceLongTaskTiming' in window) {
-          var g = (window.__tti = { e: [] });
-          g.o = new PerformanceObserver(function (l) {
-            g.e = g.e.concat(l.getEntries());
-          });
-          g.o.observe({ entryTypes: ['longtask'] });
-        }
-      })();
-    </script>
```

Comment:
this API is essentially chrome-only: https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API

pretty much anything that can be ascertained from this is already blatantly obvious by running basic a pageload profile.